idaes/surrogate/pysmo/polynomial_regression.py

```diff
@@ -949,20 +949,18 @@ def results_generation(self, beta, order):
         """
         This function prints the results of the fitting to the screen.
         """
-        results_df = pd.Series()
+        results_df = pd.Series(dtype='float64')
         counter = 1
         print("\n------------------------------------------------------------")
         print("The final coefficients of the regression terms are: \n")
         print("k               |", beta[0, 0])
-        results_df = results_df.append(pd.Series({"k": beta[0, 0]}))
+        results_df = pd.concat([results_df, pd.Series({"k": beta[0, 0]})])
```

Comment:
I would recommend being more explicit and specifying the `axis` kwarg to `pd.concat()`.

Reply:
Done